src/chttpd/src/chttpd_show.erl

```diff
@@ -204,7 +204,7 @@ handle_view_list(Req, Db, DDoc, LName, {ViewDesignName, ViewName}, Keys) ->
     DbName = couch_db:name(Db),
     {ok, VDoc} = ddoc_cache:open(DbName, <<"_design/", ViewDesignName/binary>>),
     CB = fun list_cb/2,
-    QueryArgs = couch_mrview_http:parse_params(Req, Keys),
+    Args = couch_mrview_http:parse_body_and_query(Req, Keys),
```

Comment:
I'd not change the variable name here which will shorten the diff. I assume that was done because its now more than the query string. Though we refer to view reads as queries in multiple places so there's no harm leaving it as QueryArgs.

Reply:
Fixed in 5f163849a236b9309dbb06e7f0f62e57127207d3